.changeset/spotty-flies-join.md

```diff
@@ -0,0 +1,5 @@
+---
+'@commercetools/sdk-middleware-auth': minor
```

Comment:
You need to make an actual change in the package, otherwise it won't work.

I would suggest to not mix it up with this PR though.